code/controllers/Processes/shuttles/supply.dm

```diff
@@ -16,9 +21,6 @@
 	height = 7
 	roundstart_move = "supply_away"
 
-/obj/docking_port/mobile/supply/New()
```

Comment:
Was a joke, really, Mark =p

@tigercat2000  said "it does nothing". I said "they're not goggles" in reference to "the goggles do nothing!"